tests/integration/devfile/cmd_devfile_create_test.go

```diff
@@ -115,8 +115,7 @@ var _ = Describe("odo devfile create command tests", func() {
 
 	Context("When executing odo create with devfile component type argument and --project flag", func() {
 		It("should successfully create the devfile component", func() {
-			componentNamespace := helper.RandString(6)
-			helper.CmdShouldPass("odo", "create", "java-openliberty", "--project", componentNamespace)
+			helper.CmdShouldPass("odo", "create", "java-openliberty", "--project", cmpName)
```

Comment:
We're passing component name to `--project` flag. Is that intentional?

Reply:
Yes.

Reply:
Ack, thanks. :+1: